internal/venice-client-common/src/main/java/com/linkedin/venice/stats/AbstractVeniceStats.java

```diff
@@ -38,6 +41,15 @@ public AbstractVeniceStats(MetricsRepository metricsRepository, String name) {
     // the name of total stats is usually "total" but for kafka consumer service, it is
     // "total_kafka_consumer_service_for_<region>"
     this.isTotalStats = name.equals(STORE_NAME_FOR_TOTAL_STAT) || name.startsWith(STORE_NAME_FOR_TOTAL_STAT + "_");
+
+    if (metricsRepository instanceof VeniceMetricsRepository) {
+      VeniceMetricsRepository veniceMetricsRepository = (VeniceMetricsRepository) metricsRepository;
+      VeniceMetricsConfig veniceMetricsConfig = veniceMetricsRepository.getVeniceMetricsConfig();
+      this.isTehutiMetricsEnabled = veniceMetricsConfig.emitTehutiMetrics();
+    } else {
+      this.isTehutiMetricsEnabled = true;
+    }
+    this.noopSensor = isTehutiMetricsEnabled ? null : metricsRepository.getNoopSensor("noopSensor");
```

Comment:
I don't seem to find `getNoopSensor` in my code base. Is this in the new tenuti lib?

Reply:
Yes. Added it via https://github.com/tehuti-io/tehuti/pull/37